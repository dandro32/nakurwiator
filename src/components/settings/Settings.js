import React from 'react';
import Dashboard from '../UI/dashboard/Dashboard';

const Settings = () => {
  const cards = [
    {
      title: 'Og√≥lne',
      desc: 'Mieszej w konfiguracji tej apki',
      href: '/settings/general',
      hrefTitle: 'Mieszej se',
      imgSrc: '/img/settings_general.jpg'
    }
  ];
  return (<Dashboard cards={cards} />);
};

export default Settings;