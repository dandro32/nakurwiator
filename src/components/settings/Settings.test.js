import React from 'react';
import Settings from './Settings';
import TestRenderer from 'react-test-renderer';

jest.mock('../shared/dashboard/Dashboard', () => 'Dashboard component');

describe('Settings component tests', ()=> {
  let result;

  beforeEach(() => {
    result = TestRenderer.create(
      <Settings />
    ).toTree();

  });

  it('renders correctly', () => {
    expect(result).toMatchSnapshot();
  });
});