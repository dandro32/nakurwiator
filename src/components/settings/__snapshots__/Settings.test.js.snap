// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Settings component tests renders correctly 1`] = `
Object {
  "instance": null,
  "nodeType": "component",
  "props": Object {},
  "rendered": Object {
    "instance": null,
    "nodeType": "host",
    "props": Object {
      "cards": Array [
        Object {
          "desc": "Mieszej w konfiguracji tej apki",
          "href": "/settings/general",
          "hrefTitle": "Mieszej se",
          "imgSrc": "/img/settings_general.jpg",
          "title": "Og√≥lne",
        },
      ],
    },
    "rendered": Array [],
    "type": "Dashboard component",
  },
  "type": [Function],
}
`;
